.azure-pipelines/build-template.yml

```diff
@@ -108,9 +108,19 @@ jobs:
         fi
 
         make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) target/sonic-${{ parameters.platform }}.bin
+        false
```

Comment:
when build succeed, build exit with false. why?

Reply:
this is check failure cause to see the code is working or not.

https://dev.azure.com/mssonic/build/_build/results?buildId=9374&view=results